analytics_data_api/v0/serializers.py

```diff
@@ -241,3 +241,31 @@ class Meta(object):
         model = models.CourseActivityWeekly
         # TODO: Add 'posted_forum' here to restore forum data
         fields = ('interval_start', 'interval_end', 'course_id', 'any', 'attempted_problem', 'played_video', 'created')
+
+
+class VideoSerializer(ModelSerializerWithCreatedField):
+    # pipeline_video_id is renamed to video_id
+    video_id = serializers.CharField(required=True)
+
+    class Meta(object):
+        model = models.Video
+        fields = (
+            'video_id',
+            'encoded_module_id',
+            'duration',
+            'segment_length',
+            'start_views',
+            'end_views',
+            'created'
+        )
+
+
+class VideoTimelineSerializer(ModelSerializerWithCreatedField):
+    class Meta(object):
+        model = models.VideoTimeline
+        fields = (
+            'segment',
```

Comment:
Ah, good catch.  I think the former seems to make more sense to me.  We're binning views into buckets/segments.  If we changed the field to seconds, it would seems to imply that we are counting the number of views and users at that particular discrete time point.